src/vma/dev/ib_ctx_handler.cpp

```diff
@@ -110,9 +110,19 @@ ib_ctx_handler::ib_ctx_handler(struct ib_ctx_handler_desc *desc) :
 	g_p_event_handler_manager->register_ibverbs_event(m_p_ibv_context->async_fd,
 						this, m_p_ibv_context, 0);
 
+#ifdef DEFINED_DPCP
+	m_p_adapter = set_dpcp_adapter();
```

Comment:
Will always be null

Reply:
It is set at https://github.com/Mellanox/libvma/pull/869/files#diff-6ceae2fb65065ce9a603a7e379e2d63fR297

Reply:
you are right, will fix